src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -1,17 +1,37 @@
 import { Box, Button, Separator, Serif } from "@artsy/palette"
 import React from "react"
-import { createFragmentContainer, graphql } from "react-relay"
-import { ArtworkSidebarSizeInfoFragmentContainer as SizeInfo } from "./ArtworkSidebarSizeInfo"
+import {
+  commitMutation,
+  createFragmentContainer,
+  graphql,
+  RelayProp,
+} from "react-relay"
 
 import { ArtworkSidebarCommercial_artwork } from "__generated__/ArtworkSidebarCommercial_artwork.graphql"
+import { ArtworkSidebarCommercialOrderMutation } from "__generated__/ArtworkSidebarCommercialOrderMutation.graphql"
+import { ErrorModal } from "Components/Modal/ErrorModal"
+import { ErrorWithMetadata } from "Utils/errors"
+import { ArtworkSidebarSizeInfoFragmentContainer as SizeInfo } from "./ArtworkSidebarSizeInfo"
 
 export interface ArtworkSidebarCommercialProps {
   artwork: ArtworkSidebarCommercial_artwork
+  relay?: RelayProp
+}
+
+export interface ArtworkSidebarCommercialState {
+  isCommittingCreateOrderMutation: boolean
+  isErrorModalOpen: boolean
 }
 
 export class ArtworkSidebarCommercial extends React.Component<
-  ArtworkSidebarCommercialProps
+  ArtworkSidebarCommercialProps,
+  ArtworkSidebarCommercialState
 > {
+  state = {
```

Comment:
Cool! I was looking to see where the `isCommittingCreateOrderMutation` state might be getting used, and saw there'll be a loading state/spinner on the `Button`, makes sense 👍